Installation guide and README reorganization. (#11)
intel/owca: docs/wrappers.rst

========
Wrappers
========

**This software is pre-production and should not be deployed to production servers.**

> Wrappers allow to log Application Performance Metrics from to Kafka by parsing output of an application.

Reviewer: ```suggestion
Wrappers produce Application Performance Metrics by parsing an output of an application. The metrics are stored in Kafka.
```

Output can be parsed using regular expression or a Python function.
Metrics can be normalized and labelled with user-provided values.

Wrapping ``stress-ng`` based workload example
=============================================

> #. Please follow `developemnt guide <development.rst>`_ to install depedencies.

Reviewer: s/developemnt/development/g

Author: fixed

> #. Please follow `developemnt guide <development.rst>`_ to install depedencies.

Author: ```suggestion
#. Please follow `development guide <development.rst>`_ to install dependencies.
```

#. Please follow `developemnt guide <development.rst>`_ to install depedencies.

#. Build wrappers::

    tox -e wrapper_package

#. Install stress-ng (for example using ``epel`` repository)::

    sudo yum install -y stress-ng

#. Run generic wrapper based on regexp configured for stress-ng::

    dist/wrapper.pex --command '-c "while true; do stress-ng --cpu 1 --metrics 1 --timeout 1; done"' \
        --stderr 1 --log_level DEBUG --separator ".*dispatching hogs.*" \

>         --regexp '(?P<name>cpu)[\ ]+(?P<value>\d+)' \

Author: fixed now it is `stress_ng_bogo_ops_cpu` using --metric_name_prefix

btw, the types of returned metrics were also fixed
